build.gradle.kts

```diff
@@ -141,33 +141,13 @@ val profilerDistribution = artifacts.add("archives", tasks.distZip.flatMap { it.
 
 publishing {
     publications {
-        register<MavenPublication>("mavenJava") {
-            from(components["java"])
+        named<MavenPublication>("mavenJava") {
             artifact(profilerDistribution)
-            pom {
-                licenses {
-                    license {
-                        name.set("The Apache License, Version 2.0")
-                        url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
-                    }
-                }
-            }
-        }
-    }
-
-    repositories {
-        maven {
-            name = "GradleBuildInternal"
-            url = gradleInternalRepositoryUrl()
-            credentials {
-                username = project.findProperty("artifactoryUsername") as String?
-                password = project.findProperty("artifactoryPassword") as String?
-            }
         }
     }
 }
 
-fun Project.gradleInternalRepositoryUrl(): URI {
+fun Project.gradleInternalRepositoryUrl(): java.net.URI {
```

Comment:
Any reason why we don't use the import here any more?

Reply:
This is an IDEA auto-format issue, will fix.